src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx

```diff
@@ -37,48 +44,73 @@ export const ModalAddMembers = ({
   team,
 }: ModalAddMembersProps) => {
   const { t } = useTranslation();
-  const [selectedMembers, setSelectedMembers] = useState<OptionSelect>([]);
+  const [selectedMembers, setSelectedMembers] = useState<OptionsSelect>([]);
   const [selectedRole, setSelectedRole] = useState<Role>(Role.MEMBER);
   const { toast } = useToastProvider();
   const { mutateAsync: createInvitation } = useCreateInvitation();
+  const { mutateAsync: createTeamAccess } = useCreateTeamAccess();
 
   const handleValidate = async () => {
-    const promisesMembers = selectedMembers.map((selectedMember) => {
-      return createInvitation({
-        email: selectedMember.value.email,
-        role: selectedRole,
-        teamId: team.id,
-      });
+    const promisesMembers = selectedMembers.map(async (selectedMember) => {
+      try {
+        selectedMember.type === 'invitation'
```

Comment:
nit: I feel hard-coded literals are quite dangerous, if someone refactor the hard-coded `'invitation'` value in the `SearchMembers` component, this will break

Reply:
[fixup! ✨(app-desk) add new member to team](https://github.com/numerique-gouv/people/pull/132/commits/1fd4de4bac9b15ccc9d13c921da9833cddc9b62d)